config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every 1.day, at: '8:30 am' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.